compat/types.h

```diff
@@ -49,15 +51,6 @@
 /* Typedefs for missing entries on a system           */
 /******************************************************/
 
-/*
- * Ensure that standard type limits are defined for use
- */
-#if __cplusplus
-#include <cstdint>
-#elif HAVE_STDINT_H
-#include <stdint.h>
-#endif
```

Comment:
This removal makes sense to me: We have (most likely) included inttypes.h above so we do not need to include stdint.h here.